README.md

```diff
@@ -105,22 +109,26 @@ Download/Clone the Git repository.
 
 
 ## Setup Interfaces
-Install dependencies for the interfaces, from the root of this repository, run:
+Install dependencies for the interfaces, from the individual interface directories (`interfaces/*`) run:
```

Comment:
```suggestion
Install all dependencies for all interfaces. From the root of this repository, run:
```

Reply:
@elwinschmitz  I wrote this because it was observed that when installing from root folder, repeatedly I have observed errors such as:
`Local package.json exists, but node_modules missing, did you mean to install?`